arangod/Graph/Steps/ClusterProviderStep.cpp

```diff
@@ -24,6 +24,29 @@ ClusterProviderStep::ClusterProviderStep(VertexType v, EdgeType edge,
       _edge(std::move(edge)),
       _fetched(fetched) {}
 
+ClusterProviderStep::ClusterProviderStep(VertexType v, EdgeType edge,
```

Comment:
License disclaimer is missing in this file!

Reply:
Thanks! done.